### PR TITLE
Added documentation for NOT NULL constraint.

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: blackbox/docs/sql/ddl.txt [hunk 1]
```diff
@@ -245,6 +245,24 @@ Or using a alternate syntax::
...
+
+    cr> create table my_table1 (
+    ...   first_column integer primary key,
+    ...   second_column string not null
+    ... );
+    CREATE OK, 1 row affected (... sec)
+
+.. note::
+
+    Currently not null constraint cannot be added once data is inserted.
```
Comment (mfussenegger) [inline blackbox/docs/sql/ddl.txt]: Please remove the `Currently`

Comment (matriv) [inline blackbox/docs/sql/ddl.txt]: Thought that in the future we'll support it...

File: blackbox/docs/sql/reference/copy_from.txt [hunk 2]
```diff
@@ -41,7 +41,8 @@ For examples see: :ref:`importing_data`.
   When importing data using ``COPY FROM`` Crate does not check if the types
   from the columns and the types from the import file match. It does not cast
   the types to their target but will always import the data as it is
-  in the source file.
+  in the source file. Crate does not also check for null values for columns
```
Comment (mfussenegger) [inline blackbox/docs/sql/reference/copy_from.txt]: Maybe formulate this in a more generic way that column constraints aren't checked?

File: blackbox/docs/sql/reference/create_table.txt [hunk 3]
```diff
@@ -116,6 +116,14 @@ The only table constraints currently supported are:
  * :ref:`primary_key_constraint`
  * :ref:`index-constraint`
 
+Column Constraints
+~~~~~~~~~~~~~~~~~~
+
+Column constraints are constraints that are applied on each column of the table separately.
+The only column constraints currently supported are:
```
Comment (mfussenegger) [inline blackbox/docs/sql/reference/create_table.txt]: Would formulate it as "The supported column constraints are":

**Pull request closed without merging**